mappings/net/minecraft/predicate/entity/TypeSpecificPredicate.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_7376 net/minecraft/predicate/entity/TypeSpecificPredicate
 	FIELD field_38723 ANY Lnet/minecraft/class_7376;
-	METHOD method_22494 toJson ()Lcom/google/gson/JsonObject;
+	METHOD method_22494 toJsonObject ()Lcom/google/gson/JsonObject;
```

Comment:
```suggestion
	METHOD method_22494 typeSpecificToJson ()Lcom/google/gson/JsonObject;
```
`method_22494` serializes type specific data whereas `method_43098` adds the `type` field to complete serialization. Having `toJsonObject` and `toJson` is confusing imo.